src/backend/src/common/constants/gov-ministries.constant.ts

```diff
@@ -3,77 +3,102 @@ import { GovMinistriesEnum } from '../enums/gov-ministries.enum';
 export const GovMinistries = {
   [GovMinistriesEnum.ADVANCED_EDUCATION_AND_SKILLS_TRAINING]: {
     label: 'Advanced Education and Skills Training',
+    code: 'AEST',
   },
   [GovMinistriesEnum.AGRICULTURE_AND_FOOD]: {
     label: 'Agriculture and Food',
+    code: 'AGRI',
   },
   [GovMinistriesEnum.ATTORNEY_GENERAL]: {
     label: 'Attorney General',
+    code: 'AG',
   },
   [GovMinistriesEnum.BC_PUBLIC_SERVICE_AGENCY]: {
     label: 'BC Public Service Agency',
+    code: 'PSA',
   },
   [GovMinistriesEnum.CHILDREN_AND_FAMILY_DEVELOPMENT]: {
     label: 'Children and Family Development',
+    code: 'CFD',
   },
   [GovMinistriesEnum.CITIZENS_SERVICES]: {
     label: "Citizens' Services",
+    code: 'CITZ',
   },
   [GovMinistriesEnum.EDUCATION_AND_CHILD_CARE]: {
     label: 'Education and Child Care',
+    code: 'ECC',
   },
   [GovMinistriesEnum.ENERGY_MINES_AND_LOW_CARBON_INNOVATION]: {
     label: 'Energy, Mines and Low Carbon Innovation',
+    code: 'EMLI',
   },
   [GovMinistriesEnum.ENVIRONMENT_AND_CLIMATE_CHANGE_STRATEGY]: {
     label: 'Environment and Climate Change Strategy',
+    code: 'ENV',
   },
   [GovMinistriesEnum.FINANCE]: {
     label: 'Finance',
+    code: 'FIN',
   },
   [GovMinistriesEnum.FORESTS]: {
     label: 'Forests',
+    code: 'FOR',
   },
   [GovMinistriesEnum.GOVERNMENT_COMMUNICATIONS_AND_PUBLIC_ENGAGEMENT]: {
     label: 'Government Communications and Public Engagement',
+    code: 'GCPE',
   },
   [GovMinistriesEnum.HEALTH]: {
     label: 'Health',
+    code: 'HLTH',
   },
   [GovMinistriesEnum.INDIGENOUS_RELATIONS_AND_RECONCILIATION]: {
     label: 'Indigenous Relations and Reconciliation',
+    code: 'IRR',
   },
   [GovMinistriesEnum.JOBS_ECONOMIC_RECOVERY_AND_INNOVATION]: {
     label: 'Jobs, Economic Recovery and Innovation',
+    code: 'JERI',
```

Comment:
This ministry is now 'Jobs, Economic **Development** and Innovation' with the acronym 'JEDI'